BackofficeBundle/Resources/public/coffee/ajaxLoader/AjaxLoaderView.coffee

```
@@ -0,0 +1,54 @@
+###*
+ * @namespace OpenOrchestra:AjaxLoader
+###
+window.OpenOrchestra or= {}
+window.OpenOrchestra.AjaxLoader or= {}
+
+###*
+ * @class AjaxLoaderView
+###
+class OpenOrchestra.AjaxLoader.AjaxLoaderView extends Backbone.View
+
+  tagName : 'div'
+
+  className: 'oo-ajax-loader'
+
+  ###*
+   * required options
+   * {
+   *  listenElement : {object} Jquery Element
+   * }
+   * @param {Object} options
+  ###
+  initialize: (options) ->
+    viewContext = @
+    @hideLoader(@);
```

you could also throw an exception if the required parameters is not given